app/models/feedback.rb

```diff
@@ -43,8 +36,8 @@ def is?(type)
 
   def save
     return false unless valid?
-    ZendeskSender.send!(self) unless is_feedback_with_empty_comment?
-    true
+    return save_feedback if feedback?
+    save_bug_report
```

Comment:
As discussed, is this the time to split the bug report out entirely to its own controller and model so that we can isolate the two entirely?

Reply:
We also discussed separating the feedback and bug report into different mvc.
I don't think it would be a quick task, nor was it part of the task (unforeseen) and therefore out of scope.
I would welcome a new task to action this separation.

Thoughts?

Reply:
I have created a ticket - CFP-282